yarn.lock

```diff
@@ -9,13 +9,20 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.8.3":
+"@babel/code-frame@^7.0.0":
```

Comment:
Is there a reason for the dependency upgrades? Not that it matters so much with this repo, but I'd prefer to upgrade all (possibly to latest) with a separate PR.

Reply:
Well this is interesting. I don't even have this change here on my local branch.

Reply:
I'll fix this. Thanks for pointing it out.